gateway/README.md

```diff
@@ -301,6 +302,35 @@ For example, with the config below, `documentId` will be replaced with the prope
         }
     } 
 
+##### Removing Span Tag Data
+
+If a tag contains sensitive data that needs to be removed, `RemoveSpanTags` can be used to define tags that should be removed from specific services/operations. 
+Services and Operations can be specified using a `*` for a wildcard match. If the Service or Operation field is left blank, `*` will be used as a default value. Tags must be specified using the exact name of the tag.
+For example, in the config below, the gateway will remove the "password" tag from any span that has a service that starts with "auth" and has an operation name "login". 
+It will also remove the "zipcode", "number", and "CVV" tags from any span that contains "credit-card" in the operation name, from ANY service.
+
+Use caution when leveraging this feature: every expression will impact the throughput of the Gateway. Make sure to monitor your Gateway's resource utilization and size your instance accordingly to support your needs.
+
+    {
```

Comment:
Prefer an explicit block with backticks so you can also specify the syntax highlight:

    ```json
    {
      "Type": "signalfx",
      ...
    }
    ```